examples/ui-demo/src/hooks/useRecurringTransactions.ts

```diff
@@ -174,7 +180,49 @@ export const useRecurringTransactions = (clientOptions: {
 
     // Start by minting the required USDC amount, and installing the session key, if not already installed.
 
-    const currentEpochTimeSeconds = Math.floor(Date.now() / 1000);
+    let installData: Hex | undefined;
+    if (!sessionKey.isInstalled) {
+      const currentEpochTimeSeconds = Math.floor(Date.now() / 1000);
+
+      const _client = client.extend(deferralActions);
+
+      const { entityId, nonce } = await _client.getEntityIdAndNonce({
+        entityId: sessionKey.entityId,
+        isGlobalValidation: sessionKey.isGlobalValidation,
+      });
+
+      const _installData = await new PermissionBuilder({
```

Comment:
taking this one step at a time. adding use of `PermissionBuilder` first, then planning to try to actually use the new hooks to create the session & send the user ops instead.